src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetMaxByteCount.cs

```diff
@@ -2,27 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class ASCIIEncodingGetMaxByteCount
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
> I am still thinking it would be better to fix RandomDataGenerator to produce real random data (forcing it to use random seed). 

If we do that, it should be in certain places where we expect it'll have value, and it should be planned to cover a specific set of scenarios where we have concerns that our coverage is incomplete.  For example, fuzzing.  I'm not convinced any of these tests are actually doing such things (even if there wasn't a fixed seed).